datafusion/core/tests/data/partitioned_csv/partition-0.csv

```diff
@@ -0,0 +1,11 @@
+0,0,true
```

Comment:
I captured this data by copying the result of running the code in https://github.com/apache/arrow-datafusion/blob/4a33f2708fe8e30f5f9b062a097fc6155f2db2ce/datafusion/core/tests/sql/partitioned_csv.rs#L47-L73 

I think it is easier to understand what is going on if the files are checked in rather than auto generated